requirements/base.in

```diff
@@ -1,7 +1,7 @@
 # Core requirements for using this application
 -c constraints.txt
 
-Django>=2.2,<2.3                           # Web application framework
+Django                                     # Web application framework
```

Comment:
You need to add common constraints in the `constraints.txt` otherwise `Django==3.*` will be fetched in the `base.txt` when the `make upgrade` will run.